packages/flutter/lib/widgets.dart

```diff
@@ -15,6 +15,7 @@
 library widgets;
 
 export 'package:vector_math/vector_math_64.dart' show Matrix4;
+export 'package:characters/characters.dart';
```

Comment:
Is there a better place to export this?

Reply:
This seems like the best option in lib/*.dart